docs/toolings/pchain_cli/transaction.md

```diff
@@ -49,10 +49,11 @@ Here are some CLI subcommands to indicate corresponding Protocol Transaction Com
     ```
 
 
-First, provide the following 4 parameters:
+First, decide the version of the transaction that you are creating with the flags `--v1` and `--v2`, then provide the following 4 parameters:
```

Comment:
Should be 5 params now instead of 4?

Reply:
was thinking if the "parameters" are referring to the fields in <>?

Reply:
Ah my bad, now I get what you mean - setting the flag (v1 | v2), and then setting the 4 other params. Further down it seemed like the two terms are used interchangeably: `with the flag --destination`, so maybe could be a a little clearer?